src/corner_kick/src/containers/Logger/LogItem.tsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * This file specifies the implementation of a logger item
+ */
+
+import * as React from 'react';
+
+import { IRosoutMessage, RosoutLevel } from 'SRC/types';
+import styled from 'SRC/utils/styled-components';
+
+/**
+ * Specifies most of the styling for the LogItem
+ * It also includes level dependent styling.
+ */
+const Wrapper = styled('div')<{ level: number }>`
+    width: 100%;
+
+    padding: 0 16px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    border-bottom: 1px solid ${(props) => props.theme.colors.panel};
+
+    font-size: 12px;
+
+    user-select: text;
+
+    ${(props) => {
+        switch (props.level) {
+            case RosoutLevel.DEBUG:
+                return `color: ${props.theme.colors.subdued};`;
+            case RosoutLevel.INFO:
+                return `color: ${props.theme.colors.fg};`;
+            case RosoutLevel.WARN:
+                return `color: ${props.theme.colors.warn};`;
+            case RosoutLevel.ERROR:
+                return `color: ${props.theme.colors.error};`;
+            case RosoutLevel.FATAL:
+                return `
+                    color: ${props.theme.colors.error};
+                    font-weight: 900;
+                `;
+            default:
+                return '';
```

Comment:
Hmm, we should probably have some default formatting? In case we get levels outside those we expect?

Reply:
Yup, done.